Expensify/App
# Add Workspace Policy Avatars

### Details
Adds the ability to give a workspace an avatar. Slight refactor of the Profile settings page avatar.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/169741

### Tests
### QA Steps
1. Navigate to `/settings/profile`
2. Verify there is now a pencil icon next to the Avatar
3. Tap the pencil icon and verify a menu appears
4. Test adding an avatar and removing an avatar
5. Create a new workspace
6. Navigate to the workspace page via Settings
7. Verify the pencil icon next to the Avatar is clickable
8. Tap and update the avatar
9. Verify the image updates
10. Tap the pencial again and remove the avatar
11. Verify the default avatar shows again

### Tested On


### Screenshots


#### Web
![2021-07-08_09-22-02](https://user-images.githubusercontent.com/32969087/124979200-74287e80-dfce-11eb-9bb4-48c7f447e2e9.png)
![2021-07-08_09-23-17](https://user-images.githubusercontent.com/32969087/124979206-75f24200-dfce-11eb-9890-3fbe1d0647c1.png)
![2021-07-08_09-24-37](https://user-images.githubusercontent.com/32969087/124979211-77236f00-dfce-11eb-9ee5-c22ad51e4ac0.png)
![2021-07-08_09-24-48](https://user-images.githubusercontent.com/32969087/124979216-78ed3280-dfce-11eb-8969-1af8b6f543d4.png)

#### Mobile Web
![2021-07-08_09-41-41](https://user-images.githubusercontent.com/32969087/124981028-cd91ad00-dfd0-11eb-8a48-5b6d857bb752.png)
![2021-07-08_09-41-54](https://user-images.githubusercontent.com/32969087/124981035-cec2da00-dfd0-11eb-8fc2-f0bbf4132fc7.png)
![2021-07-08_09-41-54](https://user-images.githubusercontent.com/32969087/124981041-d08c9d80-dfd0-11eb-8080-ebc2386f9eea.png)
![2021-07-08_09-41-57](https://user-images.githubusercontent.com/32969087/124981047-d2566100-dfd0-11eb-964f-ea8f7123cd6e.png)
![2021-07-08_09-42-13](https://user-images.githubusercontent.com/32969087/124981049-d4202480-dfd0-11eb-93f9-3866824bb5ba.png)

#### Desktop
![2021-07-08_09-56-27](https://user-images.githubusercontent.com/32969087/124982792-d5525100-dfd2-11eb-824d-1cebf504cfc1.png)
![2021-07-08_09-56-35](https://user-images.githubusercontent.com/32969087/124982798-d6837e00-dfd2-11eb-96ed-39d4b60e1839.png)
![2021-07-08_09-57-49](https://user-images.githubusercontent.com/32969087/124982906-ff0b7800-dfd2-11eb-8ab5-32edc99c8462.png)

#### iOS
![2021-07-08_09-33-57](https://user-images.githubusercontent.com/32969087/124980854-9e7b3b80-dfd0-11eb-84d7-27a381cb72f3.png)
![2021-07-08_09-34-01](https://user-images.githubusercontent.com/32969087/124980859-9fac6880-dfd0-11eb-9fea-ad2b7ad2ccda.png)
![2021-07-08_09-34-06](https://user-images.githubusercontent.com/32969087/124980865-a0dd9580-dfd0-11eb-811b-7b678997b81e.png)
![2021-07-08_09-40-22](https://user-images.githubusercontent.com/32969087/124980872-a20ec280-dfd0-11eb-92a7-75f30a5553a4.png)

#### Android
![2021-07-08_10-24-36](https://user-images.githubusercontent.com/32969087/124985966-c40b4380-dfd6-11eb-9a14-e973100400a1.png)
![2021-07-08_10-24-41](https://user-images.githubusercontent.com/32969087/124985971-c53c7080-dfd6-11eb-804c-9a7bf5c662c9.png)
![2021-07-08_10-24-50](https://user-images.githubusercontent.com/32969087/124985981-c8cff780-dfd6-11eb-8cc6-f12513d61c71.png)
![2021-07-08_10-24-56](https://user-images.githubusercontent.com/32969087/124985987-ca99bb00-dfd6-11eb-8fb2-28698b46be7f.png)
